SConstruct

```diff
@@ -443,3 +450,40 @@ else:
     for x in platform_list:
         print("\t" + x)
     print("\nPlease run scons again with argument: platform=<string>")
+
+
+screen = open('/dev/tty', 'w')
```

Comment:
Jep, noticed it in the Travis check. Is `sys.stdout` available across the platforms?